docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+
+version: '2'
+
+services:
+  tsr1.yul1.example.net:
+    image: "internap/fake-switches:latest"
+    environment:
+      - SWITCH_MODEL=cisco_2960_48TT_L
+    ports:
+      - "32769:22"
+
+  netman:
+    image: "internap/netman:latest"
```

Comment:
shouldn't the docker compose file references the dockerfile on the local checkout of netman ?
https://docs.docker.com/compose/compose-file/#dockerfile

Or we could let it like that ?